### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by rspt**
View commit list.

If it's ok for you, then you can merge it and deploy it :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html lang="fr-CH">
-<head>
+<head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# business: http://ogp.me/ns/business#">
```
Comment (maxirozay) [inline index.html]: What is it for? Is it to create the customized google results?

File: src/router.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import VueRouter from 'vue-router'
 
 const Home = resolve => require(['./components/Home.vue'], resolve)
 const HowItWorks = resolve => require(['./components/HowItWorks.vue'], resolve)
-const Vehicle = resolve => require(['./components/Vehicle.vue'], resolve)
+const Vehicule = resolve => require(['./components/Vehicule.vue'], resolve)
```
Comment (maxirozay) [inline src/router.js]: Why do you write this one in french?

**Comment by maxirozay** [comment]
You didn't create a static page for every page you only added the title and description in every page?

**Comment by maxirozay** [comment]
Good job!!! I uploaded already and I'll merge when you change the title of the vehicle component or when you tell me why is should be vehicule.

File: src/router.js (hunk 2 above)
Comment (rspt) [inline src/router.js]: Rollbacked ;)

**Comment by rspt** [comment]
> You didn't create a static page for every page you only added the title and description in every page?

Finally no, too much works to do using Nuxt. But you can check what I've done in the nuxt branch of this repo.

> Good job!!! I uploaded already and I'll merge when you change the title of the vehicle component or when you tell me why is should be vehicule.

Thanks :)

Rolled back!

**Pull request merged**